src/actions/onboardingActions.js

```diff
@@ -128,7 +114,7 @@ export const setupUserAction = (username: ?string) => {
     if (isOnline) {
       logBreadcrumb('onboarding', 'setupUserAction: user is online, registering for FCM Remote Notifications');
       // we us FCM notifications so we must register for FCM, not regular native Push-Notifications
-      await firebaseMessaging.registerForRemoteNotifications().catch((error) => {
+      await firebaseMessaging.registerDeviceForRemoteMessages().catch((error) => {
         reportErrorLog('firebaseMessaging.registerForRemoteNotifications failed', { error });
```

Comment:
Maybe change the error log message to firebaseMessaging.registerDeviceForRemoteMessages failed since the function name is changed?

Reply:
We'll sort this out in the next release.